index.html

```diff
@@ -1,221 +1,213 @@
-          </h4>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul>
-            <li>html</li>
-            <li>CSS</li>
-            <li>javascript</li>
-          </ul>
-          <button class="button">See Project</button>
-        </div>
-      </article>
-      <article class="card">
-        <div class="cardContainer">
-          <img
-            class="img"
-            src="images/SnapshootPortfolio4.png"
-            alt="First Image"
-          />
-          <h3>Multi-Post Stories</h3>
-          <h4>
-            <span>CANOPY</span>&nbsp;
-            <img src="images/Counter.png" alt="counter" />&nbsp; Back End
-            Dev&nbsp; <img src="images/Counter.png" alt="counter" />&nbsp; 2015
-          </h4>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul>
-            <li>html</li>
-            <li>CSS</li>
-            <li>javascript</li>
-          </ul>
-          <button class="button">See Project</button>
-        </div>
-      </article>
-      <article class="about">
-        <section class="aboutsection">
-          <div class="aboutContainer">
-            <div id="primary-text-about">About Myself</div>
-            <div id="secondary-text-about">
-              Hello I’m a software developer! I can help you build a product ,
-              feature or website Look through some of my work and experience! If
-              you like what you see and have a project you need coded, don’t
-              hestiate to contact me.
-            </div>
-            <div class="connect">LET'S CONNECT</div>
-            <div class="socialmedia">
-              <a href="https://github.com/sajjadrajaee">
-                <img src="images/social/Github.svg" alt="social icons" />
-              </a>
-              <a href="https://linkedin.com/in/sajjadrajaee">
-                <img src="images/social/Linkedin.svg" alt="social icons" />
-              </a>
-              <a href="#">
-                <img src="images/social/Vector.svg" alt="social icons" />
-              </a>
-              <a href="https://twitter.com/sajjadrajaee">
-                <img src="images/social/twitter.svg" alt="social icons" />
-              </a>
-              <a href="#">
-                <img src="images/social/M.svg" alt="social icons" />
-              </a>
-            </div>
-            <button class="button">Get My Resume</button>
+          <div id="connect">LET'S CONNECT</div>
+          <div id="socialmedia">
+            <a href="https://twitter.com/sajjadrajaee">
+              <img src="images/social/twitter.svg" alt="social icons" />
+            </a>
+            <a href="https://linkedin.com/in/sajjadrajaee">
+              <img src="images/social/Linkedin.svg" alt="social icons" />
+            </a>
+            <a href="#">
```

Comment:
- [ ] Kindly don't forget to add a link to your social as this `a` currently takes you back to the home page.